src/css/controls/flags/overlay.less

```diff
@@ -5,9 +5,5 @@
         .jw-logo {
             display: none;
         }
-
-        .jw-gradient {
```

Comment:
@robwalch this was put because when overlay is open, if you look very carefully you might notice a slight gradient. I'll move it up to the display none block on top